backend/src/routes/__test__/person.route.test.ts

```diff
@@ -57,13 +132,365 @@ describe('person ', () => {
 
     // update a person
     await supertest(app)
-        .put(`/api/persons/${newPersonId}`)
-        .set('Accept', 'application/json')
-        .send(requestPersonData)
-        .expect(httpStatus.NO_CONTENT);
+      .put(`/api/persons/${newPersonId}`)
+      .set('Accept', 'application/json')
+      .send(requestPersonData)
+      .expect(httpStatus.NO_CONTENT);
 
     // retrieve encounter from database, and check that the updated encounter contains the changed location
     const resPerson = await supertest(app).get(`/api/persons/${newPersonId}`);
     expect(resPerson._body.location).toBe(requestPersonData.location);
   });
 });
+
+describe('GET /persons', () => {
+  it('Response paginated and returns correct number of entries', async () => {
+    const person1 = new Person(person1Data);
```

Comment:
Same comment as on the encounter tests, could we extract this to a method?

Reply:
Yep, will do